kyo-core/shared/src/main/scala/kyo/fibers.scala

```diff
@@ -125,7 +125,9 @@ object Fibers extends Joins[Fibers] with fibersPlatformSpecific:
 
     case object Interrupted
         extends RuntimeException("Fiber Interrupted")
-        with NoStackTrace
+        with NoStackTrace:
+        override def getCause() = null
```

Comment:
Surprising this is not the default behavior

Reply:
Yeah, it'd be nice if `NoStrackTrace` had an override for this method.